debian/changelog

```diff
@@ -1,4 +1,4 @@
-cloud-init (23.3-0ubuntu0~23.04.1) UNRELEASED; urgency=medium
+cloud-init (23.3-0ubuntu1~23.04.1) UNRELEASED; urgency=medium
```

Comment:
Let's drop this changeset as it will be replaced with 23.3.1-0ubuntu1~23.04.1 on our next SRU anyway